### PR TITLE
New 'validate file paths' action in Gold Images Processing command

### PR DESCRIPTION
**Pull request by josecelano**
I've migrated the media file location validation.

### CODE REVIEW

File: src/nautilus_librarian/typer/commands/workflows/actions/validate_filenames.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def validate_filenames(dvc_diff):
     if dvc_diff == "{}":
         return ActionResult(ResultCode.EXIT, [Message("No Gold image changes found")])
 
-    filenames = extract_modified_media_file_list_from_dvd_diff_output(dvc_diff)
+    filenames = extract_list_of_media_file_changes_from_dvd_diff_output(dvc_diff)
```
Comment (yeraydavidrodriguez) [inline src/nautilus_librarian/typer/commands/workflows/actions/validate_filenames.py:22]: Please fix typo: dvC_diff_output

File: src/nautilus_librarian/typer/commands/workflows/actions/validate_filenames.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 from nautilus_librarian.mods.dvc.domain.utils import (
-    extract_modified_media_file_list_from_dvd_diff_output,
+    extract_list_of_media_file_changes_from_dvd_diff_output,
```
Comment (yeraydavidrodriguez) [inline src/nautilus_librarian/typer/commands/workflows/actions/validate_filenames.py:2]: Please fix typo: dvC_diff_output

File: tests/test_nautilus_librarian/test_typer/test_commands/test_workflows/test_actions/test_message.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from nautilus_librarian.typer.commands.workflows.actions.action_result import Message
+
+
+def test_messages_can_be_compared():
+    msg1 = Message("message 1")
+    msg2 = Message("message 1")
+
```
Comment (yeraydavidrodriguez) [inline tests/test_nautilus_librarian/test_typer/test_commands/test_workflows/test_actions/test_message.py:7]: I would remove this new line between msg2 and msg3 declarations

File: tests/test_nautilus_librarian/test_mods/test_dvc/test_domain/test_utils.py [hunk 4]
```diff
@@ -2,11 +2,11 @@
 
 from nautilus_librarian.mods.dvc.domain.utils import (
     extract_added_files_from_dvc_diff,
-    extract_modified_media_file_list_from_dvd_diff_output,
+    extract_list_of_media_file_changes_from_dvd_diff_output,
```
Comment (yeraydavidrodriguez) [inline tests/test_nautilus_librarian/test_mods/test_dvc/test_domain/test_utils.py:5]: Please fix typo: dvC_diff_output

File: tests/test_nautilus_librarian/test_mods/test_dvc/test_domain/test_utils.py [hunk 5]
```diff
@@ -2,11 +2,11 @@
...
 )
 
 
-def test_extract_modified_media_file_list_from_dvd_diff_output():
+def test_extract_list_of_media_file_changes_from_dvd_diff_output():
```
Comment (yeraydavidrodriguez) [inline tests/test_nautilus_librarian/test_mods/test_dvc/test_domain/test_utils.py:9]: Please fix typo: dvC_diff_output

**Review by yeraydavidrodriguez — CHANGES_REQUESTED** [review]
Reviewed, some minor typos to be fixed

File: tests/test_nautilus_librarian/test_mods/test_dvc/test_domain/test_utils.py (hunk 5 above)
Comment (josecelano) [reply tests/test_nautilus_librarian/test_mods/test_dvc/test_domain/test_utils.py]: Fixed

File: tests/test_nautilus_librarian/test_mods/test_dvc/test_domain/test_utils.py (hunk 4 above)
Comment (josecelano) [reply tests/test_nautilus_librarian/test_mods/test_dvc/test_domain/test_utils.py]: Fixed

File: tests/test_nautilus_librarian/test_typer/test_commands/test_workflows/test_actions/test_message.py (hunk 3 above)
Comment (josecelano) [reply tests/test_nautilus_librarian/test_typer/test_commands/test_workflows/test_actions/test_message.py]: Fixed

File: src/nautilus_librarian/typer/commands/workflows/actions/validate_filenames.py (hunk 2 above)
Comment (josecelano) [reply src/nautilus_librarian/typer/commands/workflows/actions/validate_filenames.py]: Fixed

File: src/nautilus_librarian/typer/commands/workflows/actions/validate_filenames.py (hunk 1 above)
Comment (josecelano) [reply src/nautilus_librarian/typer/commands/workflows/actions/validate_filenames.py]: Fixed

**Review by yeraydavidrodriguez — APPROVED** [review]

**Pull request merged**